tests.sh

```diff
@@ -61,39 +105,38 @@ echo "hostname:"
 echo $HOSTNAME
 echo
 echo "DAOS_DIR:"
-BUILD=`ls -ald $(realpath $DAOS_DIR/../.)`
+BUILD=`ls -ald $(realpath ${DAOS_DIR}/../.)`
 echo $BUILD
 
 mkdir -p ${RUN_DIR}
-cp -v ${DAOS_DIR}/../repo_info.txt ${RUN_DIR}
+cp -v ${DAOS_DIR}/../repo_info.txt ${RUN_DIR}/repo_info_${SLURM_JOB_ID}.txt
+cat ${RUN_DIR}/repo_info_${SLURM_JOB_ID}.txt
 
 source ${DST_DIR}/env_daos ${DAOS_DIR}
-source ${DST_DIR}/build_env.sh ${MPI}
+source ${DST_DIR}/build_env.sh ${MPI_TARGET}
 
-export PATH=$DAOS_DIR/install/ior_$MPI/bin:$PATH
-export LD_LIBRARY_PATH=$DAOS_DIR/install/ior_$MPI/lib:$LD_LIBRARY_PATH
+export PATH=${DAOS_DIR}/install/ior_${MPI_TARGET}/bin:${PATH}
+export LD_LIBRARY_PATH=${DAOS_DIR}/install/ior_${MPI_TARGET}/lib:${LD_LIBRARY_PATH}
 
 echo PATH=$PATH
 echo
 echo LD_LIBRARY_PATH=$LD_LIBRARY_PATH
 echo
 
-echo TESTCASE=$TESTCASE
-echo DAOS_SERVERS=$DAOS_SERVERS
-echo DAOS_CLIENTS=$DAOS_CLIENTS
-echo PPC=$PPC
-echo RANKS=$no_of_ps
-echo OCLASS=$OCLASS
-
 # Generate timestamp
 function time_stamp(){
     date +%m/%d-%H:%M:%S
 }
 
-# Print message, timestap is prefixed
+# Print message, timestamp is prefixed
 function pmsg(){
     echo
-    echo "$(time_stamp) ${1}"
+    echo "$(time_stamp) ${@}"
+}
+
+function pmsg_err(){
+    echo
+    echo "$(time_stamp) ERR ${@}"
```

Comment:
I'd like to eventually have a standard message for something like:
"FAIL_REASON: Test failed because ...."
Then we can parse this from the logs to see WHY the test failed